h2o-core/src/main/java/water/fvec/NewChunk.java

```diff
@@ -21,7 +21,7 @@
 // An uncompressed chunk of data, supporting an append operation
 public class NewChunk extends Chunk {
 
-  private static final boolean DEBUG_SAFE_BUFX = H2O.getSysBoolProperty("debug.safe_bufx", false); 
+  private static final boolean USE_SAFE_BUFX = H2O.getSysBoolProperty("debug.safe_bufx", true); 
```

Comment:
maybe you also want to change the sys property name as well: no one is aware of the current one anyway.

Reply:
but the purpose is still for debugging, right? maybe even more so now than before

Reply:
is it? I don't see what additional information it provides, and how it can help to figure out what is the issue with the unsafe version.
If you mean `debug` as "removing a bug", then yeah (it's a fix then), but `debug` as "trying to identify a bug" then for me it's not what it's doing.

Reply:
I mean it in the sense that it is for "debugging"; no user should really be setting that flag.

I see it as a mechanism to revert the behavior back when I am debugging the old code. WDYT?

I can rename but my pattern is to use "debug" for all stuff that should be enabled/disabled in development mode.

Reply:
@michalkurka I see what you mean now. Agree that it's only for dev anyway, so I was not about to argue longer :)